src/Service/VendorService/VendorCoreService.php

```diff
@@ -222,9 +229,9 @@ public function processBatch(array $batch, SourceRepository $sourceRepo, string
 
         foreach ($batch as $identifier => $imageUrl) {
             if (array_key_exists($identifier, $sources)) {
-                if ($withUpdates) {
-                    /* @var Source $source */
-                    $source = $sources[$identifier];
+                /* @var Source $source */
+                $source = $sources[$identifier];
+                if ($source->getDate() >= $withUpdatesDate && $source->getSearches()->count() < 1) {
```

Comment:
Won't `$source->getSearches()->count()` always be at least `1` if we always index the identifier given by the vendor?

Reply:
Fixed